app/javascript/components/enrollment/steps/ExposureInformation.js

```diff
@@ -0,0 +1,561 @@
+              type="switch"
+              id="healthcare_personnel"
+              label="HEALTHCARE PERSONNEL"
+              checked={this.state.current.patient.healthcare_personnel || false}
+              onChange={this.handleChange}
+            />
+          </Form.Group>
+          <Form.Group as={Col} md="auto" className="mb-0 my-auto ml-4">
+            <Form.Control
+              size="sm"
+              className="form-square"
+              id="healthcare_personnel_facility_name"
+              placeholder="enter facility name"
+              value={this.state.current.patient.healthcare_personnel_facility_name || ''}
+              onChange={this.handleChange}
+              aria-label="Enter Healthcare Facility Name"
+            />
+            <Form.Control.Feedback className="d-block" type="invalid">
+              {this.state.errors['healthcare_personnel_facility_name']}
+            </Form.Control.Feedback>
+          </Form.Group>
+        </Form.Row>
+        <Form.Row>
+          <Form.Group as={Col} md="auto" className="mb-0 my-auto pb-2">
+            <Form.Check
+              className="pt-2 my-auto"
+              type="switch"
+              id="crew_on_passenger_or_cargo_flight"
+              label="CREW ON PASSENGER OR CARGO FLIGHT"
+              checked={this.state.current.patient.crew_on_passenger_or_cargo_flight || false}
+              onChange={this.handleChange}
+            />
+          </Form.Group>
+        </Form.Row>
+        <Form.Row>
+          <Form.Group as={Col} md="auto" className="mb-0 my-auto pb-2">
+            <Form.Check
+              className="pt-2 my-auto"
+              type="switch"
+              id="member_of_a_common_exposure_cohort"
+              label="MEMBER OF A COMMON EXPOSURE COHORT"
+              checked={this.state.current.patient.member_of_a_common_exposure_cohort || false}
+              onChange={this.handleChange}
+            />
+          </Form.Group>
+          <Form.Group as={Col} md="auto" className="mb-0 my-auto ml-4">
+            <Form.Control
+              size="sm"
+              className="form-square"
+              id="member_of_a_common_exposure_cohort_type"
+              placeholder="enter description"
+              value={this.state.current.patient.member_of_a_common_exposure_cohort_type || ''}
+              onChange={this.handleChange}
+              aria-label="Enter Cohort Description"
+            />
+            <Form.Control.Feedback className="d-block" type="invalid">
+              {this.state.errors['member_of_a_common_exposure_cohort_type']}
+            </Form.Control.Feedback>
+          </Form.Group>
+        </Form.Row>
+        {!this.props.currentState.isolation && (
+          <Form.Row>
+            <Form.Group as={Col} md="24" controlId="exposure_notes" className="pt-3 mb-2">
+              <Form.Label className="input-label">NOTES{schema?.fields?.exposure_notes?._exclusive?.required && ' *'}</Form.Label>
```

Comment:
The ticket calls for the notes to be included
> - When the Exposure Information is edited, the "Notes" section should be displayed but the "Public Health Risk Assessment and Management" section should be hidden.
>   - Add a tooltip next to the "Notes" section in both the Case Status and Exposure information screens referencing that it is a shared notes field between both screens (validate the language with the PO team). Also, update the placeholder text for the notes field so it reads "enter additional information" without referencing exposure or case info.

Was it decided somewhere not to do this at some point?

Reply:
It was discussed on @mdube7's original PR [here](https://github.com/SaraAlert/SaraAlert/pull/975#issuecomment-875800064) and the decision was ultimately made by @hackrm

Reply:
The reasoning is because if you updated it for exposure info, then in case info, it was confusing why the original note was overwritten (because they're the same field)